examples/pytorch/gatv2/gatv2.py

```diff
@@ -0,0 +1,55 @@
+"""
+Graph Attention Networks in DGL using SPMV optimization.
+References
+----------
+Paper: https://arxiv.org/abs/1710.10903
+Author's code: https://github.com/PetarV-/GAT
+Pytorch implementation: https://github.com/Diego999/pyGAT
+"""
+
+import torch
+import torch.nn as nn
+import dgl.function as fn
+from dgl.nn import GATv2Conv
+
+
+class GATv2(nn.Module):
+    def __init__(self,
+                 g,
```

Comment:
Can you make `g` an argument of `forward`. `GATv2Conv` takes `g` as an input argument anyway. We will update the GAT example you adapted.